cluster-configuration/services-configuration.yaml

```diff
@@ -34,27 +34,27 @@ cluster:
   docker-registry-info:
 
     # If public, please fill it the same as your username
-    docker-namespace: your_registry_namespace
+    docker-namespace: openpai
 
     # E.g., gcr.io. If public，fill docker_registry_domain with word "public"
     # docker_registry_domain: public
-    docker-registry-domain: your_registry_domain
```

Comment:
If user wanna use public registry, they could get it from our document. If you concern that they may miss this infomation, they could generated the configuration, and the default registry is docker.io. I don't think we should set this field as docker.io and leave username and password as empty.

Reply:
in the readme.md, we have clearly explain all these. I assume the four yaml files are examples, not template. 

---
In reply to: [195305097](https://github.com/Microsoft/pai/pull/727#discussion_r195305097) [](ancestors = 195305097)